src/data/request.ts

```diff
@@ -48,29 +48,35 @@ export const requestData = async (options: IRequestDataOptions): Promise<ICaseVa
       return match;
     });
 
-  const cases = filteredRows
-    .reduce<Record<string, ICaseValue>>((acc, [attributeId, countryId, yearId, value]) => {
-      const attribute = attributeMap[attributeId];
+  const _countryIds = countryIds.length > 0 ? countryIds : countries.map(c => c.id);
```

Comment:
You could probably move `countries.map(c => c.id)` and `years.map(y => y.id)` to be a invariant constant outside this function.

Reply:
Done in the last commit.